d2l-tspan.html

```diff
@@ -69,18 +90,51 @@
 			attached: function() {
 				var slot = Polymer.dom(this.root).querySelector('#float');
 				this._sizeObserver.observe(slot);
+				slot.addEventListener('focusout', this._boundFocusOutHandler);
 			},
 			detached: function() {
 				var slot = Polymer.dom(this.root).querySelector('#float');
 				this._sizeObserver.unobserve(slot);
 			},
 			_selectedChanged: function(selected) {
 				this.setAttribute('aria-selected', selected.toString());
+				var slot = Polymer.dom(this.root).querySelector('#float');
+				slot.removeEventListener('focusout', this._boundFocusOutHandler);
 			},
 			_onSizeChanged: function() {
 				if (this.root.host) {
 					this.root.host.style.height = this.$.float.offsetHeight + 'px';
 				}
+			},
+			focus: function() {
+				if (this.customBorder && !this.feedbackInFocus) {
+					this.feedbackInFocus = true;
+					this.addBorderToFloat();
+				}
+			},
+			_focusOutHandler: function(event) {
+				if (this.customBorder) {
+					this.feedbackInFocus = false;
+					this.removeBorderFromFloat();
+				}
+			},
+			addBorderToFloat: function() {
+				if (this.customBorder) {
+					var float = Polymer.dom(this.root).querySelector('#float');
```

Comment:
What's the rationale for rendering these styles inline?

Note: CSS colors variables should be used.

Reply:
Using this to change the border when the component is hovered or focussed. I can't use the focus selector since the textarea-input inside the d2l-rubric-feedback component is focused to capture text.
CSS color variables did not work with polymer1 when I used them inline. Worked with polymer3 though and my PR for master is using color variables.

Reply:
I think you should be able to move these style properties into CSS, and add/remove a class.  That would allow you to use the color variables, although I think it is also possible to use them from JavaScript since on occasion we do override them there.  It doesn't need to be applied via `:focus` pseudo-selectors in order to do that.  Perhaps I am misunderstanding?

Reply:
Updated this and the private property name change as well!

Reply:
Is this actually necessary? When using the component, could you not just put `tspan[focusesd]` before your styling for the stuff in the tspan so that the focus style is only applied when it is a descendant of a focused tspan element. Then you wouldn't need to manually add and remove classes in the tspan element.

Reply:
I think I see what you mean.
`focused` is an attribute/property that will tell the tspan if its being used for feedback for rubrics and so if custom styling is needed when hovered/pretend-focused.
`_feedbackInFocus` is the property that is being used to tell if it's being hovered or if its being pretend-focused (i.e. do I remove the styling when the cursor moves off or keep it).
Refactored accordingly!